.github/CODEOWNERS

```diff
@@ -2,5 +2,6 @@
 docs/eigenlayer/ @eigensid @teddyknox @dabit3 @wesfloyd @tinahaibodi @bpm333 @joan-el @scotthconner @QuinnLee
 docs/eigenlayer/security @eigensid @teddyknox @dabit3 @wesfloyd @tinahaibodi @ropcode @antojoseph @anupsv @wadealexc
 docs/eigenlayer/avs-guides/ @eigensid @teddyknox @dabit3 @wesfloyd @tinahaibodi @bpm333 @joan-el @scotthconner @QuinnLee @NimaVaziri @afkbyte @stevennevins @samlaf @0xkydo
-docs/eigenlayer/risk/ @0xkydo @eigensid @teddyknox @dabit3 @wesfloyd @tinahaibodi @cryptogakusei 
+docs/eigenlayer/risk/ @0xkydo @eigensid @teddyknox @dabit3 @wesfloyd @tinahaibodi @cryptogakusei
```

Comment:
can you also clean the codeowners file here? remove eigensid and tinehaibodi and potentially ropcode (have no idea who that is)